denizen_scripts/behrcraft/repo-link/Commands/Essentials/Essentials Home.dsc

```diff
@@ -216,17 +216,7 @@ Home_GUI:
             - inventory open d:<[Inventory]>
 
         - case ModeSelect:
-            - choose <[Mode]>:
-                - case Teleport:
-                    - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/Teleport].escaped>
-                - case Rename:
-                    - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/Rename].escaped>
-                - case Relocate:
-                    - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/Relocate].escaped>
-                - case HomeSelect:
-                    - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/HomeSelect].escaped>
-                - case Delete:
-                    - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/Delete].escaped>
+            - run Home_GUI def:<list[Menu/Home_GUI|Action/Main_Menu|Mode/<[Mode]>].escaped>
```

Comment:
This was a good change - but this list can be passed as a single list entry using `<list_single[<list>]>` like this:
`- run Home_GUI def:<list_single[<list[Menu/Home_GUI|Action/Main_Menu|Mode/<[Mode]>]>]>`